plugins/modules/lambda.py

```diff
@@ -634,9 +650,9 @@ def main():
         response = get_current_function(client, name, qualifier=current_version)
         if not response:
             module.fail_json(msg='Unable to get function information after updating')
-
+        response = format_response(response)
         # We're done
-        module.exit_json(changed=changed, **camel_dict_to_snake_dict(response))
+        module.exit_json(changed=changed, **response)
```

Comment:
In theory, yes.

However, I prefer splitting it out because we have two places where we're making the same transformation.  By turning it into a function we reduce the chance if introducing inconsistent behaviour in a future change.

For example: When making this change I initially missed that this transformation was happening in two different spots!